docs/rules/complexity.md

```diff
@@ -57,7 +59,7 @@ Optionally, you may specify a `maximum` object property:
 is equivalent to
 
 ```json
-"complexity": [2, {"maximum": 2}]
+"complexity": [2, { "maximum": 2} ]
```

Comment:
I see a space after opening brace, no space before closing brace, and space before closing square bracket (with no space after opening square bracket). Could you make this more consistent please?